frontend/components/Navbar.jsx

```diff
@@ -18,28 +18,35 @@ const globalMenuItems = [
       { title: "Manifesto", path: "/manifesto", reactive: false },
     ],
   },
-  { title: "Occurrences", path: "/occurrences", reactive: false },
-  { title: "Quotes", path: "/quotes", reactive: false },
+  {
+    title: "Occurrences",
+    path: "/search/?content_types=occurrences.occurrence",
+    as: "/occurrences",
+    reactive: true
+  },
+  {
+    title: "Quotes",
+    path: "/search/?content_types=quotes.quote",
+    as: "/quotes",
+    reactive: true
+  },
```

Comment:
Allows routing to `/occurrences` and `/quotes` without leaving the Next app using decorator urls (the `as` argument), though these routes do not exist yet, so refreshing routes to Django.